rest/data/perf.go

```diff
@@ -233,6 +237,31 @@ func (dbc *DBConnector) FindPerformanceResultWithChildren(ctx context.Context, i
 	return apiResults, nil
 }
 
+// ScheduleSignalProcessingRecalculateJobs schedules signal processing recalculation jobs for
+// each project/version/task/test combination
+func (dbc *DBConnector) ScheduleSignalProcessingRecalculateJobs(ctx context.Context) error {
+	queue := dbc.env.GetRemoteQueue()
+	ids, err := model.GetTimeSeriesIds(ctx, dbc.env)
+	if err != nil {
+		return gimlet.ErrorResponse{StatusCode: http.StatusInternalServerError, Message: fmt.Sprint("Failed to get time series ids")}
+	}
+	for _, id := range ids {
+		job := units.NewRecalculateChangePointsJob(id)
+		err = queue.Put(ctx, job)
+		if err != nil {
+			return errors.Wrap(err, message.WrapError(err, message.Fields{
```

Comment:
I would errors.New() this. 

also I would consider doing a continue on error kind of thing here, to avoid cases where a collision early on blocks later jobs. 

we can talk about changing the logging/error reporting strategy.

Reply:
Changed to errors.New(). Used `grip.NewBasicCatcher` here as well to continue on error.